gui/default/syncthing/folder/editFolderModalView.html

```diff
@@ -11,11 +11,22 @@ <h4 class="modal-title">
         <form role="form" name="folderEditor">
           <div class="row">
             <div class="col-md-12">
+              <div class="form-group" ng-class="{'has-error': folderEditor.folderLabel.$invalid && folderEditor.folderLabel.$dirty}">
+                <label for="folderLabel"><span translate>Folder Label</span></label>
+                <input name="folderLabel" id="folderLabel" class="form-control" type="text" ng-model="currentFolder.label" ng-if="editingExisting"/>
+                <input name="folderLabel" id="folderLabel" class="form-control" type="text" ng-model="currentFolder.label" ng-if="!editingExisting &&  currentFolder.viewFlags.importFromOtherDevice" value="{{currentFolder.label}}"/>
```

Comment:
Quick explanation of why we have the the duplication and if on importFromOtherDevice here? I'm sure there's a good reason, it's just not obvious to me when reading this. :)